functions/submit-finding.js

```diff
@@ -35,12 +37,44 @@ async function getContestEnd(contestId) {
   return new Date(contest.end_time).getTime();
 }
 
+async function updateTeamData(team, newPolygonAddress) {
+  // @todo: delete this once all existing teams have added addresses
+  const updatedTeamData = JSON.stringify(
+    {
+      ...team,
+      address: newPolygonAddress,
+    },
+    null,
+    2
+  );
+  const files = {
+    [`_data/handles/${teamName}.json`]: updatedTeamData,
+  };
+  const body = `This auto-generated PR adds polygon address for team ${teamName}`;
+  const title = `Add address for team ${teamName}`;
+  await octokit.createPullRequest({
+    owner: "code-423n4",
+    repo: "code423n4.com",
+    title,
+    body,
+    head: `warden/${teamName}`,
```

Comment:
Make sure to add this to the ignore script (and/or nest another layer to ignore)

Reply:
Oh, good catch. I misread the ignore script and thought it just checked for `warden` but now I see it looks for `warden-`